randovania/games/fusion/logic_database/Sector 2 TRO.txt

```diff
@@ -1840,7 +1849,8 @@ Hint Features - Multiple Pickups
 > Inside Puzzle; Heals? False
   * Layers: default
   > Pickup (Energy Tank)
-      Trivial
+      # Softlock risk
+      Morph Ball or Knowledge (Beginner)
```

Comment:
i'm really not sure about this. you can in all circumstances leave the exact same way you entered in. and gating "just turn around" behind knowledge feels weird

Reply:
Id be fine having "morph or movement (beginner) to reach the top item

> maybe some of the vanilla spots should be counted as tricks someday

Very unlikely. Almost all people who run rando know how to do the tricks, and those who dont and feel intimidated by them can turn them off as location spots.
Making these tricks would have a lot of whacky side effects